Library/ViewModels/ConfirmDetailsViewModel.swift

```diff
@@ -224,13 +230,81 @@ public class ConfirmDetailsViewModel: ConfirmDetailsViewModelType, ConfirmDetail
       context.map { $0.confirmationLabelHidden }
     )
 
-    self.configurePledgeSummaryViewControllerWithData = Signal.combineLatest(
+    let pledgeTotalSummaryData = Signal.combineLatest(
       projectAndConfirmationLabelHidden,
       pledgeTotal
     )
     .map(unpack)
     .map { project, confirmationLabelHidden, total in (project, total, confirmationLabelHidden) }
     .map(pledgeSummaryViewData)
+
+    self.configurePledgeSummaryViewControllerWithData = pledgeTotalSummaryData
+
+    // MARK: Pledge + Rewards Summary Table
+
+    self.pledgeRewardsSummaryViewHidden = Signal.zip(context, baseReward)
```

Comment:
Is `zip` here correct?

Reply:
I believe so since this output only needs to emit based on the context and baseReward one time.